sql/core/src/main/scala/org/apache/spark/sql/internal/CatalogImpl.scala

```diff
@@ -365,7 +365,7 @@ class CatalogImpl(sparkSession: SparkSession) extends Catalog {
   override def dropTempView(viewName: String): Boolean = {
     sparkSession.sessionState.catalog.getTempView(viewName).exists { viewDef =>
       sparkSession.sharedState.cacheManager.uncacheQuery(
-        sparkSession, viewDef, cascade = false, blocking = true)
```

Comment:
Let's just take out this argument, and ultimately make a non-blocking call.

Reply:
ah, ok.

Reply:
I just noticed that `uncacheQuery` has `blocking = true` by default, so I changed it then took out the argument here.